frontend/src/services/api/examples.ts

```
@@ -0,0 +1,138 @@
+    });
+    
+    return response.data;
+  } catch (error) {
+    if (error.name === 'AbortError') {
+      console.log('Search request was cancelled');
+    }
+    throw error;
+  }
+  
+  // To cancel: controller.abort()
+  return { controller };
```

Biome (1.9.4)

[error] This code is unreachable

... because either this statement will return from the function ...

... or this statement will throw an exception beforehand


(lint/correctness/noUnreachable)